src/components/Maincontainer.css

```diff
@@ -1,275 +1,31 @@
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  font-weight: 1000;
-  font-family: "Baloo Paaji", cursive;
-  color: #695aa6;
+@media only screen and (max-width: 767px) {
+  .hlBpIo {
+    padding-left: 11%;
+    padding-right: 3%;
+  }
+  .services-items .card {
+    margin-bottom: 14px;
+    width: 341px;
+    min-height: 106px;
+  }
+  .portfolio-card .card {
+    width: 60%;
```

Comment:
While using the `flexbox` or bootstraps `col` classes, avoid using static widths.

Reply:
Done